redshift_sqlalchemy/dialect.py

```diff
@@ -350,7 +352,8 @@ def __init__(self, schema_name, table_name, data_location, access_key, secret_ke
         self.access_key = access_key
         self.secret_key = secret_key
         self.session_token = session_token
-        self.options = options
+        self.options = options or {}
+        self.format = format
```

Comment:
well they all do don't they.

I guess CopyCommand should probably take a Table object instead.